### PR TITLE
refactor(ui): Extract searchable page into its own component (perf + ux) 

### PR DESCRIPTION
**Pull request by jjoyce0510**
This PR prevents page flickering by refactoring the SearchablePage out as a wrapper container around other existing pages 

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: datahub-web-react/src/app/glossary/BusinessGlossaryPage.tsx [hunk 1]
```diff
@@ -87,7 +86,7 @@ function BusinessGlossaryPage() {
                         )}
                     </MainContentWrapper>
                 </GlossaryWrapper>
-            </SearchablePage>
+            </>
```
Comment (chriscollins3456) [inline datahub-web-react/src/app/glossary/BusinessGlossaryPage.tsx:89]: I think you can get rid of this fragment

File: datahub-web-react/src/app/ingest/ManageIngestionPage.tsx [hunk 2]
```diff
@@ -67,6 +66,6 @@ export const ManageIngestionPage = () => {
                     {selectedTab === TabType.Sources ? <IngestionSourceList /> : <SecretsList />}
                 </ListContainer>
             </PageContainer>
-        </SearchablePage>
+        </>
```
Comment (chriscollins3456) [inline datahub-web-react/src/app/ingest/ManageIngestionPage.tsx:69]: same here

File: datahub-web-react/src/app/search/SearchablePage.tsx [hunk 3]
```diff
@@ -36,6 +36,10 @@ const defaultProps = {
  * A page that includes a sticky search header (nav bar)
  */
 export const SearchablePage = ({ initialQuery, onSearch, onAutoComplete, children }: Props) => {
```
Comment (chriscollins3456) [inline datahub-web-react/src/app/search/SearchablePage.tsx:38]: can you remove `initialQuery` as a prop now?

File: datahub-web-react/src/app/settings/SettingsPage.tsx [hunk 4]
```diff
@@ -128,6 +127,6 @@ export const SettingsPage = () => {
                     ))}
                 </Switch>
             </PageContainer>
-        </SearchablePage>
+        </>
```
Comment (chriscollins3456) [inline datahub-web-react/src/app/settings/SettingsPage.tsx:130]: can also remove here (and a few others at the top of this PR)

**Review by chriscollins3456 — APPROVED** [review]
Looks good! I have some small nits but you can ignore if you're trying to merge in asap for deployment

File: datahub-web-react/src/app/settings/SettingsPage.tsx (hunk 4 above)
Comment (jjoyce0510) [reply datahub-web-react/src/app/settings/SettingsPage.tsx:130]: yep!

**Pull request merged**